packages/devtools_app/lib/src/shared/device_dialog.dart

```diff
@@ -62,9 +62,7 @@ class DeviceDialog extends StatelessWidget {
         'Framework / Engine': '${flutterVersion!.frameworkRevision} / '
             '${flutterVersion!.engineRevision}',
       },
-      if (serviceManager.service!.connectedUri != null)
-        'VM Service Connection':
-            serviceManager.service!.connectedUri.toString(),
+      'VM Service Connection': serviceManager.service!.connectedUri.toString(),
```

Comment:
instead of removing the if check, check that `serviceManager.service?.connectedUri != null`

Reply:
done